### PR TITLE
Fixed test suite execution when having legacy debugging tests

### PR DESCRIPTION
**Pull request by kevinmessiaen**
## Description

Fixed test suite execution when having legacy debugging tests

## Type of Change

- [ ] 📚 Examples / docs / tutorials / dependencies update
- [x] 🔧 Bug fix (non-breaking change which fixes an issue)
- [ ] 🥂 Improvement (non-breaking change which improves an existing feature)
- [ ] 🚀 New feature (non-breaking change which adds functionality)
- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🔐 Security fix

## Checklist

- [ ] I've read the [`CODE_OF_CONDUCT.md`](https://github.com/Giskard-AI/ai-inspector/blob/master/CODE_OF_CONDUCT.md) document.
- [ ] I've read the [`CONTRIBUTING.md`](https://github.com/Giskard-AI/ai-inspector/blob/master/CONTRIBUTING.md) guide.
- [ ] I've updated the code style using `make codestyle`.
- [ ] I've written tests for all new methods and classes that I created.
- [ ] I've written the docstring in Google format for all the methods and classes that I used.

### CODE REVIEW

File: tests/communications/test_websocket_actor_tests.py
Comment (Inokinoki) [inline tests/communications/test_websocket_actor_tests.py:1]: Do you think whether we need a test case with `my_simple_test_legacy_debug_no_name`, which does set `output_df`?

This is to make sure that the upload code in test suite running is covered.
We forgot it last time, which causes the issue.
I can do it if you agree but do not have much time ;)

Comment (kevinmessiaen) [reply tests/communications/test_websocket_actor_tests.py:1]: I can add one test RN

Comment (kevinmessiaen) [reply tests/communications/test_websocket_actor_tests.py:1]: I added the test

Comment (Inokinoki) [reply tests/communications/test_websocket_actor_tests.py:1]: Thanks very much!

**Review by Inokinoki — APPROVED** [review]
LGTM!

**Pull request merged**